GETTING_STARTED.md

```diff
@@ -0,0 +1,380 @@
+
+## 1.2. Connecting the dots
+One of the main ideas is to store enterprise data in simple tables which are interconnected — which do have relations. You could start with simple and dumb tables. If you start to connect the data in the right way, you will leverage from rich product information. This idea is not new, but every relational database (Oracle, PostgreSQL, MySQL, etc.) out there shows how powerful it can be. We took this idea and its simple core concepts and built a user & consumer centered enterprise database. The core parts of GRUD consists of an RESTful API and an easy-to-use web-based user interface.
+
+## 1.3. Editing / Publishing Separation
+Another driving force behind GRUD's software architecture is the principle "Separation of Concerns". Traditional content management systems ship both "content editing" & "content publishing" in one piece of software. Both systems are interleaved but still serve highly different use cases. Content editing and content publishing in an omnichannel universe do have totally different requirements. The life-cycle of e.g. a website is different to a central enterprise database like GRUD.
+
+# 2. Getting started
+
+> All the following URLs are relative to your API endpoint.
+> 
+> E.g. https://example.org/api
+
+## 2.1. Data retrieval
+
+The schema is structured in tables and columns. A table consists of a set of columns and of course its rows. All API endpoints are generic and therefore are not bound to your schema at all. The next three examples will show you how to traverse the data structure.
+
+### Retrieving tables
+
+`GET /tables`
+
+Call this endpoint to retrieve all tables.
+
+```
+{
+  "tables": [
+    {
+      "id": 1, // unique table ID
+      "name": "country", // internal and unique table name
+      
+      //
+      // Meta information:
+      //
+      "hidden": false,
+      "displayName": { // user-friendly table name
+        "de": "Land",
+        "en": "Country"
+      },
+      "description": { // user-friendly table description
+        "de": "Liste aller benötigten Länder"
+        // ... more languages optional
+      },
+      "group": { // tables can be grouped
+        "id": 10,
+        "displayName": {
+          "de": "Allgemein",
+          "en": "Common"
+        },
+        "description": {}
+      }
+    }
+    // ... more tables to come
+  ]
+} 
+```
+
+### Retrieving columns
+
+`GET /tables/:tableid/columns`
+
+Call this endpoint of a specific table to retrieve its columns. The different data type will be exampled later. Each table consists of at least one column. Additionally at least one of the tables columns does have the `identifier` flag. The `identifier` flag is used to distinguish one row to another. Multiple `identifier` columns are possible. The `identifier` value of a row is be used as a link value — this will be example in the data types chapter. 
```

Comment:
Multiple columns may have an identifier flag, which are used to distinguish one row from another. At least one column must be set as identifier. <- Are no identifier columns impossible? I thought the first column would be used, if none was set...

The identifier value of a row is used in link values - there will be an example in the data types chapter.

Reply:
the different data type**s** will be **explained** later
distinguish [..] **from** [...]
the `identifier` [..] **can/might** be used...